tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -90,6 +90,13 @@ BusType = PCIe
 PCI.BusNum = 0x0
 PCI.DevNum = 0x0
 
+[DAQmxDevice Dev1]
```

Comment:
Can we give appropriate name for device? `Dev1` seems to be generic

Reply:
Updated device name.